snuba/cli/consumer.py

```diff
@@ -93,6 +93,9 @@
 @click.option(
     "--profile-path", type=click.Path(dir_okay=True, file_okay=False, exists=True)
 )
+@click.option(
+    "--parallel-collect", is_flag=True, default=False,
+)
```

Comment:
nit: I find this more readable when the order here matches below - can you put this above the optional ones?

Reply:
Done